massa-consensus-worker/src/state/prune.rs

```diff
@@ -12,7 +17,8 @@ impl ConsensusState {
     /// prune active blocks and return final blocks, return discarded final blocks
     fn prune_active(&mut self) -> Result<PreHashMap<BlockId, ActiveBlock>, ConsensusError> {
         // list required active blocks
-        let mut retain_active = self.list_required_active_blocks()?;
+        let mut retain_active: PreHashSet<BlockId> =
+            self.list_required_active_blocks()?.into_iter().collect();
```

Comment:
I think this `into_iter().collect()` is useless

Reply:
Indeed, leftovers of a previous version

Reply:
Done